### PR TITLE
Bug 1135805 - making chrome urlbar visible to the screen reader only if ...

### PR DESCRIPTION
**Pull request by yzen**
...it is visible onscreen.

### CODE REVIEW

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=e80d3a8d3ff6)

File: apps/system/style/chrome/chrome.css [hunk 1]
```diff
@@ -216,7 +216,17 @@
...
-  transition: transform 0.2s ease-in-out;
+  transition: transform 0.2s ease-in-out, visibility 0s;
+  visibility: visible;
+}
+
+[data-manifest-name="FTU"]:not(.bar) .controls .urlbar,
+.lockScreenWindow:not(.bar) .controls .urlbar,
+.homescreen:not(.bar) .controls .urlbar,
+.globalOverlayWindow:not(.bar) .controls .urlbar,
+.appWindow.fullscreen-app .chrome:not(.dragged):not(.dragging) .controls .urlbar {
```
Comment (KevinGrandon) [inline apps/system/style/chrome/chrome.css]: This patch is only for fullscreen apps, correct? If that's the case, I would much rather have a rule only for full screen apps. One for ```.fullscreen-app .chrome.dragged, .fullscreen-app .chrome.dragging``` and one for ```.fullscreen-app .chrome```. I don't really like the idea of adding all of the extra noise in the selectors here.

Is this possible?

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2733f6362954)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=7509263bcb46)

**Comment by try-server-hook** [comment]
[yzen Yura Zenevich (yzen)](https://github.com/yzen) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=913c37ed9d6a)

**Pull request merged**